future-apps/payroll/contracts/Payroll.sol

```diff
@@ -502,6 +521,17 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         emit AddEmployee(employeeId, _accountAddress, _initialDenominationSalary, _role, _startDate);
     }
 
+    /**
+     * @dev Add a requested amount to the bonus amount for a given employee
+     * @param _employeeId Employee's identifier
+     * @param _amount Amount be added to the employee's bonus amount
+     */
+    function _addBonus(uint256 _employeeId, uint256 _amount) internal {
+        Employee storage employee = employees[_employeeId];
+        employee.bonus = employee.bonus.add(_amount);
```

Comment:
Wondering if we should max the bonus and accrued value at `MAX_UINT256` as well, so everything just uses that as a hard stop?

Reply:
In this case it will be hold by SafeMath, the employer won't be able to overflow the employee's bonus amount basically. The same holds now for reimbursements (accrued value) since we split it from accrued salary. The only thing that could overflow is the owed salary, which is a concern for accrued salary and payroll payments :/